docs/miscellaneous.rst

```diff
@@ -192,6 +192,11 @@ These are regular array indices into a list of source files usually called
 ``"sourceList"``, which is part of the combined-json and the output of
 the json / npm compiler.
 
+.. note ::
+    In the case of bytecode that is not associated with any particular source file,
```

Comment:
Maybe extending it with `In the case of bytecode (sections) that` or using instructions instead of bytecode is less misleading. Having read it out of context (skipping the first two paragraphs) I thought it means that an entire bytecode is not associated with a file.

Reply:
Fair enough; I adjusted the commit slightly.